gazelle/python/testdata/relative_imports/BUILD.in

```diff
@@ -1 +1,2 @@
 # gazelle:resolve py resolved_package //package2:resolved_package
+# gazelle:python_generation_mode project
```

Comment:
Can you make these examples work in package mode? We shouldn't require project mode to use relative imports